packages/react-native-reanimated/src/worklets/native/NativeWorklets.ts

```diff
@@ -0,0 +1,29 @@
+'use strict';
+import { checkCppVersion } from '../../platform-specific/checkCppVersion';
+import { getValueUnpackerCode } from '../../valueUnpacker';
+import WorkletsModule from '../specs/NativeWorkletsModule';
+import { ReanimatedError } from '../../errors';
+
+/** Type of `__workletsModuleProxy` injected with JSI. */
+export interface NativeWorkletsModule {}
+
+export class NativeWorklets {
+  #nativeWorkletsModule: NativeWorkletsModule;
+
+  constructor() {
+    if (global.__workletsModuleProxy === undefined) {
```

Comment:
I think we also need to add a mock implementation of the module for the Web

Reply:
You are right, I forgot about it.

Reply:
I will add it in a separate PR when this mock implementation will become necessary.

Reply:
Akctchually it become needed, added in [ba033d3](https://github.com/software-mansion/react-native-reanimated/pull/6539/commits/ba033d3e891598d96a44d7ca83d832df64f62cd3)